server/serve.go

```diff
@@ -23,11 +24,38 @@ func getQuestionName(z *zones.Zone, fqdn string) string {
 	return strings.ToLower(strings.Join(ql, "."))
 }
 
+func getIPFromDomain(dashedip string) (net.IP, error) {
+	i := strings.Split(dashedip, ".")[0]
```

Comment:
Suggest a few name changes:
dashedip -> dashedIP
variable name i and p can be more verbose and meaningful

Reply:
Fixd.